lib/model/query/sessions.js

```diff
@@ -42,7 +42,9 @@ const terminate = (session) => ({ run }) =>
 terminate.audit = (session) => (log) => {
   // don't audit user logouts, since they're just normal user actions.
   if ((session.actor != null) && (session.actor.type === 'user')) return;
-  return log('session.end', session.actor);
+  const prefix = ((session.actor != null) && (session.actor.type != null))
+    ? `${session.actor.type}.` : '';
+  return log(`${prefix}session.end`, session.actor);
```

Comment:
`session.actorId` is not nullable in the database, so maybe it's possible to remove the null checks for `session.actor`?